src/app/(with-header)/mypage/activities/components/ActivityCard.tsx

```
@@ -0,0 +1,102 @@
+              <MoreOptionsIcon size={40} />
+            </button>
+
+            {isMenuOpen && (
+              <>
+                <div
+                  className='fixed inset-0 z-40'
+                  onClick={() => setIsMenuOpen(false)}
+                />
+
+                {/* 드롭다운 메뉴 */}
+                <div className='absolute top-full right-0 z-50 w-160 rounded-md border border-gray-300 bg-white shadow-lg'>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)